mlflow/cli.py

```diff
@@ -281,6 +283,36 @@ def server(backend_store_uri, default_artifact_root, host, port,
         sys.exit(1)
 
 
+@cli.command()
+@click.option("--backend-store-uri", metavar="PATH",
+              default=DEFAULT_LOCAL_FILE_AND_ARTIFACT_PATH,
+              help="URI to which to persist experiment and run data. Acceptable URIs are "
+                   "SQLAlchemy-compatible database connection strings "
+                   "(e.g. 'sqlite:///path/to/file.db') or local filesystem URIs "
+                   "(e.g. 'file:///absolute/path/to/directory'). By default, data will be deleted "
+                   "from the ./mlruns directory.")
+@click.option("--run-ids", default=None,
+              help="Optional comma separated list of runs to be permanently deleted."
+                   "If run ids are not specified, by default, all deleted runs are gc'ed.")
+@experimental
+def gc(backend_store_uri, run_ids):
+    """
+    Permanently delete runs in state deleted in the backend store.
```

Comment:
@t-henri Can we explain that this utility deletes both artifacts and run data?

Reply:
Done